src/permissions/rules/rules.service.ts

```diff
@@ -41,14 +41,24 @@ export class RulesService {
         since: this.lastSeq,
         include_docs: true,
         doc_ids: JSON.stringify([Permission.DOC_ID]),
+        // requests somehow time out after 1 minute
+        timeout: 50000,
       }),
     );
+    const before = new Date().getTime();
 
     return getParams
       .pipe(
-        concatMap((params) =>
-          this.couchdbService.get<ChangesResponse>(db, '_changes', params),
-        ),
+        concatMap((params) => {
+          const date = new Date().getTime();
+          const diff = (date - before) / 1000;
+          console.log(diff.toFixed(2), 'restarting');
```

Comment:
remove console.log before merge? Or is this useful debugging info for production systems also?